integration-cli/docker_cli_pull_test.go

```diff
@@ -30,11 +30,9 @@ func (s *DockerHubPullSuite) TestPullFromCentralRegistry(c *check.C) {
 
 	// We should have a single entry in images.
 	img := strings.TrimSpace(s.Cmd(c, "images"))
-	if splitImg := strings.Split(img, "\n"); len(splitImg) != 2 {
-		c.Fatalf("expected only two lines in the output of `docker images`, got %d", len(splitImg))
-	} else if re := regexp.MustCompile(`^hello-world\s+latest`); !re.Match([]byte(splitImg[1])) {
-		c.Fatal("invalid output for `docker images` (expected image and tag name")
-	}
+	splitImg := strings.Split(img, "\n")
+	c.Assert(len(splitImg), checker.Equals, 2, check.Commentf("expected only two lines in the output of `docker images`, got %d", len(splitImg)))
```

Comment:
@calavera yep

I think it could be :

```go
c.Assert(splitImg, checker.HasLen, 2, check.Commentf("expected only two lines in the output of `docker images`, got %d", img))
```